include/xlnt/utils/numeric.hpp

```diff
@@ -176,7 +176,8 @@ class number_serialiser
         }
         char buf[30];
         assert(s.size() < sizeof(buf));
```

Comment:
The assertion should now be `assert(s.size() + 1 < sizeof(buf));`. Just in case :wink:

Reply:
Thanks, I will change it.